tests/qprogram/test_qblox_compiler.py

```diff
@@ -388,6 +389,7 @@ def test_no_loops_all_operations(self, no_loops_all_operations: QProgram):
                             wait             40
                             wait             100
                             play             0, 1, 1000
+                            set_mrk          7
```

Comment:
only for my personal knowledge, the number set in the QASM is the base 10 translation of the base 2 mask we are setting, right? Hence the 7 for 0111?

Reply:
Exactly.